filters/addbackground.go

```diff
@@ -17,31 +17,43 @@ func NewAddBackground() filters.Spec {
 	return &addBackground{}
 }
 
-func (r *addBackground) Name() string {
+func (s *addBackground) Name() string {
 	return AddBackgroundName
 }
 
-func (r *addBackground) CreateOptions(image *bimg.Image) (*bimg.Options, error) {
-	log.Debug("Create options for adding background ", r)
+func (s *addBackground) CreateOptions(image *bimg.Image) (*bimg.Options, error) {
+	log.Debug("Create options for adding background ", s)
 
 	if image.Type() != "png" {
 		return &bimg.Options{}, nil
 	}
 
-	backgroundColor := bimg.Color{R: r.R, G: r.G, B: r.B}
+	backgroundColor := bimg.Color{R: s.R, G: s.G, B: s.B}
 
 	return &bimg.Options{
 		Background: backgroundColor}, nil
 
 }
 
-func (r *addBackground) CreateFilter(args []interface{}) (filters.Filter, error) {
+func (s *addBackground) CanBeMerged(other *bimg.Options, self *bimg.Options) bool {
+	zero := bimg.Color{}
+
+	//it can be merged if the background was not set (in options or in self) or if they are set to the same value
+	return other.Background == zero || other.Background == self.Background
```

Comment:
Question: which filter takes priority.. e.g. if I set

```
addBackground(255, 0, 0) -> addBackground(0, 255, 0)
```

will the background be red or green?